libraries/standard/http/cbmc/proofs/demo/Makefile

```diff
@@ -0,0 +1,82 @@
+METHOD=IotMqtt_Disconnect
+ENTRY=harness
+
+################################################################
+# Proof assumptions
+
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# allow mqtt mallocs to fail for coverage
+DEF += -include mqtt_state.h
+DEF += -DIotMqtt_MallocMessage=malloc_can_fail
+DEF += -DIotMqtt_MallocOperation=malloc_can_fail
+
+################################################################
+
+SRCDIR ?= $(abspath $(PROOFDIR)/../../..)
+
+INC += \
+	-I$(SRCDIR)/cbmc/proofs \
+	-I$(SRCDIR)/demos \
+	-I$(SRCDIR)/libraries/ \
+	-I$(SRCDIR)/libraries/platform \
+	-I$(SRCDIR)/libraries/standard/common/include \
+	-I$(SRCDIR)/libraries/standard/mqtt/include \
+	-I$(SRCDIR)/libraries/standard/mqtt/src \
+	-I$(SRCDIR)/libraries/standard/mqtt/src/private \
+	-I$(SRCDIR)/ports/common/include \
+
+DEF += \
+        -DIOT_SYSTEM_TYPES_FILE=\"$(SRCDIR)/ports/posix/include/iot_platform_types_posix.h\" \
+        -Diotmqtt_EXPORTS \
+
+COMPILE_FLAGS = -std=gnu99
+LINK_FLAGS = -std=gnu99
+
+NONDET_STATIC=--nondet-static
```

Comment:
If this is a best practices demo, I would prefer a comment to indicate that this is optional. You've already written a comment about NONDET_STATIC in Makefile.common, but I would actually prefer to see that comment here, since people are more likely to read it. Also make it clear that it should be set to empty, rather than unset, to switch it off.